app/views/planning_applications/consultees/index.html.erb

```diff
@@ -23,15 +23,56 @@
 
       } do %>
 
+    <section class="consultee-selection govuk-!-margin-top-6">
+      <h2 class="govuk-heading-m">Select constraints that require consultation</h2>
+      <p class="govuk-body">Review the planning constraints and select which ones require consultation.</p>
+
+      <div class="govuk-checkboxes consultee-selection__list" data-module="govuk-checkboxes">
+        <% manageable_consultees = !@planning_application.determined? %>
+        <% @planning_application.planning_application_constraints.each do |constraint| %>
+          <% form_data_attributes = {} %>
+          <% form_data_attributes[:controller] = "auto-submit" if manageable_consultees %>
+
+          <%= form_with scope: :planning_application_constraint,
+                url: planning_application_consultees_assign_constraint_path(@planning_application),
+                method: :post,
+                local: true,
+                data: form_data_attributes,
+                class: "consultee-selection__form" do |constraint_form| %>
+            <%= constraint_form.hidden_field :constraint, value: constraint.id %>
+            <div class="govuk-checkboxes__item">
+              <% checkbox_options = {
+                   class: "govuk-checkboxes__input",
+                   id: dom_id(constraint, :consultation_required)
+                 }
+                 checkbox_options[:data] = {action: "change->auto-submit#submit"} if manageable_consultees
+                 checkbox_options[:disabled] = true unless manageable_consultees %>
```

Comment:
Not vital, but I think element attributes can be simplified, because they're left out if the value is nil. Which means this should work:

```suggestion
              <% checkbox_options = {
                   class: "govuk-checkboxes__input",
                   id: dom_id(constraint, :consultation_required),
                   data: manageable_consultees ? {action: "change->auto-submit#submit"} : nil,
                   disabled: !manageable_consultees
                 } %>
```

(or in fact could go inline in the `check_box_tag` call.)